include/FxLine.h

```diff
@@ -91,17 +95,22 @@ class FxLine : public QWidget
 	QColor m_strokeInnerInactive;
 	static QPixmap * s_sendBgArrow;
 	static QPixmap * s_receiveBgArrow;
-
-	QStaticText m_staticTextName;
+	bool m_inRename;
+	QString m_newName;
```

Comment:
It looks like this member is not needed. A local variable `newName` would be better.